frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue

```diff
@@ -273,8 +276,8 @@ export default class CircuitSandboxWidget extends Vue.with(Props) {
       align-items: center;
       justify-content: center;
 
-      height: 9rem;
-      width: 100%;
+      height: 16px;
```

Comment:
One question I had here — I'm wondering if these values needed to change?

Updating the `height` and `width` caused the formula area to be too small. However, if we leave them at `height: 9rem` and `width: 100%`, the formula is readable.

Please see this screen capture in Firefox 👇🏼 



https://user-images.githubusercontent.com/6276074/191624451-5f3a5e2e-609b-4185-a04e-fdd8ee2c1750.mov